ansible/roles/test/files/ptftests/py3/ip_in_ip_tunnel_test.py

```diff
@@ -24,8 +24,8 @@
 PACKET_NUM_FOR_NEGATIVE_CHECK = 100
 
 DIFF = 0.25 # The valid range for balance check
-SRC_IP_RANGE = [unicode('8.0.0.0'), unicode('8.255.255.255')]
-SRC_IPV6_RANGE = [unicode('20D0:A800:0:00::'), unicode('20D0:FFFF:0:00::FFFF')]
+SRC_IP_RANGE = [str('8.0.0.0'), str('8.255.255.255')]
```

Comment:
Are literal strings Unicode by default in Python3?

Reply:
Yes. 

Reference: https://docs.python.org/3/howto/unicode.html#the-string-type

Reply:
You could simply put like:
```
SRC_IPV6_RANGE = ['20D0:A800:0:00::', '20D0:FFFF:0:00::FFFF']
```

Reply:
updated accordingly.